x-pack/plugins/cases/server/custom_fields/number.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import Boom from '@hapi/boom';
+import { isNumber } from 'lodash';
+
+export const getCasesNumberCustomField = () => ({
+  isFilterable: true,
+  isSortable: false,
+  savedObjectMappingType: 'number', // CHECK!!!
```

Comment:
This is used for filtering right? Should the type instead be [one from these](https://www.elastic.co/guide/en/elasticsearch/reference/current/number.html)? maybe @js-jankisalvi I think you worked on this

Reply:
yeah saved object mapping type for number is `long` [x-pack/plugins/cases/server/saved_object_types/cases/cases.ts]( https://github.com/elastic/kibana/blob/main/x-pack/plugins/cases/server/saved_object_types/cases/cases.ts)